backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -321,6 +321,149 @@ class SchedulingService implements ISchedulingService {
     }
   }
 
+  async sendAdminEmailVerificationAfterDonationScheduled(
+    donor: UserDonorDTO,
+    schedule: SchedulingDTO,
+  ): Promise<void> {
+    if (!this.emailService) {
+      const errorMessage =
+        "Attempted to call sendAdminEmailVerificationAfterDonationScheduled but this instance of SchedulingService does not have an EmailService instance";
+      Logger.error(errorMessage);
+      throw new Error(errorMessage);
+    }
+
+    try {
+      const { firstName, lastName } = donor;
+      // HARD CODED TEST ADMIN EMAIL
+      const adminEmail = "jessiepeng@uwblueprint.org";
+
+      // Proposed drop off info
+      const { startTime, endTime } = schedule;
+      const startTimeToLocalDate = startTime.toLocaleString("en-US", {
+        timeZone: "EST",
+      });
+
+      const startDayString: string = dayjs(startTimeToLocalDate).format(
+        "dddd, MMMM D",
+      );
+
+      const startTimeString: string = dayjs(startTimeToLocalDate).format(
+        "h:mm A",
+      );
+      const endTimeString: string = dayjs(
+        endTime.toLocaleString("en-US", {
+          timeZone: "EST",
+        }),
+      ).format("h:mm A");
+
+      // frequency string
+      // e.g. Weekly on <day of week> until <recurringDonationEndDate>
+      let frequencyString = "One-time donation";
+      if (schedule.frequency !== Frequency.ONE_TIME) {
+        if (schedule.frequency === Frequency.DAILY) {
+          frequencyString = `Daily`;
+        } else if (schedule.frequency === Frequency.WEEKLY) {
+          frequencyString = `Weekly on ${dayjs(startTimeToLocalDate).format(
+            "dddd",
+          )}`;
+        } else {
+          frequencyString = `Monthly on the ${ordinal(
+            Number(dayjs(startTimeToLocalDate).format("D")),
+          )}`;
+        }
+      }
+
+      let volunteerTime = "";
```

Comment:
this is the _exact same code_ as our donor scheduling verification, 
can we please move this into either a utility function or a separate function in this class that accepts `startTime, endTime` as input params and returns all the formatted variables you need?

Reply:
then call on this function for both the donor email and the admin email